Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -500,7 +500,7 @@ public static void DrawSubProfileEditor(Object profileObject, bool renderProfile
             }
 
             var subProfile = profileObject as BaseMixedRealityProfile;
-            if (subProfile != null && !subProfile.IsCustomProfile)
+            if (subProfile != null && !subProfile.IsCustomProfile && MixedRealityProjectPreferences.LockProfiles)
             {
                 EditorGUILayout.HelpBox("Clone this default profile to edit properties below", MessageType.Warning);
```

Comment:
When `MixedRealityProjectPreferences.LockProfiles` is true, should we update this to warn something along the lines of 

>You're editing a default profile. These changes may be overwritten during an MRTK upgrade. We recommend cloning for editing profiles.

?